File: app/src/main/java/it/unical/mat/embasp/languages/asp/ASPFilterOption.java
package it.unical.mat.embasp.languages.asp;

import it.unical.mat.embasp.base.OptionDescriptor;

/** Generic filter option for ASP solver */

public abstract class ASPFilterOption extends OptionDescriptor {
	public ASPFilterOption() {
		super("-filter=");
	}
}

File: CSharp version/it/unical/mat/embasp/languages/asp/ASPFilterOption.cs
namespace it.unical.mat.embasp.languages.asp
{
    using OptionDescriptor = it.unical.mat.embasp.@base.OptionDescriptor;

    public abstract class ASPFilterOption : OptionDescriptor
    {
        public ASPFilterOption() : base("-filter=") { }
    }
}